jira/komand_jira/actions/delete_user/action.py

```diff
@@ -20,7 +23,8 @@ def run(self, params={}):
 
         if not self.connection.is_cloud and not params.get(Input.USERNAME):
             raise PluginException(
-                preset=PluginException.Preset.USERNAME_PASSWORD, assistance="Jira server need username to be set",
+                preset=PluginException.Preset.USERNAME_PASSWORD,
+                assistance="Jira server need username to be set",
```

Comment:
When using a `preset` argument, a custom `assistance` will not show. So you'll need to pick one or the other (I think the custom assistance is best for this scenario).

Reply:
Hey @mrinehart-r7,
I wasn't actually working on that action, I just applied black formatting to the plugin as the code wasn't formatted correctly. I've updated the `preset=` with `cause=` and added a message. Let me know if anything else needs changing 🙂

Reply:
Ah, sorry I missed that! Thank you for fixing that though :)